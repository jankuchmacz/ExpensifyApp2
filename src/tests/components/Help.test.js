import React from "react";
import HelpPage from "../../components/Help";
import { shallow } from "enzyme";

test('should render help page correctly', ()=>{
    const wrapper = shallow(<HelpPage/>)
    expect(wrapper).toMatchSnapshot();
})