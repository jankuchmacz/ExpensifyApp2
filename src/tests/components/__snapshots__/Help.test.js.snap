// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render help page correctly 1`] = `
<div>
  This is help component
</div>
`;
